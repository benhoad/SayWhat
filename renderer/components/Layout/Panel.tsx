interface IPanelProps {
  children?: any;
}

export default function Panel({ children }: IPanelProps) {
  return (
    <>
      <div className="Panel">{children}</div>

      <style jsx>{`
        .Panel {
          height: 100vh;
          display: flex;
          flex-direction: column;
        }
      `}</style>
    </>
  );
}
