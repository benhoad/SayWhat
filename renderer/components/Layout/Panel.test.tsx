import { render } from "@testing-library/react";

import Panel from "./Panel";

describe("Panel", () => {
  it("renders", () => {
    expect.hasAssertions();

    const { container } = render(<Panel>PanelContent</Panel>);
    expect(container.firstElementChild).toMatchSnapshot();
  });
});
