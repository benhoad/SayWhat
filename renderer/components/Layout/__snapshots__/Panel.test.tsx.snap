// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Panel renders 1`] = `
<div
  class="jsx-3358665321 Panel"
>
  PanelContent
</div>
`;
